session/handler.go

```diff
@@ -287,12 +287,12 @@ type listSessionsRequest struct {
 	Active bool `json:"active"`
 
 	// ExpandOptions is a query parameter encoded list of all properties that must be expanded in the Session.
-	// Example - ?expand=Identity&expand=Devices
 	// If no value is provided, the expandable properties are skipped.
 	//
 	// required: false
+	// enum: identity,devices
```

Comment:
Very good point, I did not think about that. But I think in this case it's fine because the endpoint was just introduced a few days ago to prod. In the API we use lowercase everywhere, so I think it's good to streamline this!